### ISSUE TITLE
Bug in freight-add

### ISSUE THREAD

**Issue by tumblingtumbleweed** [rcrowley/freight#38]
When a control file does not have a space following a colon, the apt.sh script removes the first character of the following field.

For example:

```
# [freight] adding oracle-jrockit-jvm_1.6.0-45-28.2.7_amd64.deb to pool (as oracle-jrockit-jvm_1.6.0-45-28.2.7_md64.deb)
tee: /disk1/apt/precise/dists/precise/main/binary-md64/Packages: No such file or directory
```

Notice "amd64" vs "md64"

**Comment by tumblingtumbleweed** [comment]
I suggest changing

```
egrep -i "^$2:" "$1" | cut -d: -f2- | cut -c2-
```

to

```
egrep -i "^$2:" "$1" | cut -d: -f2- | awk '{print $1}'
```

**Comment by edgan** [comment]
The control file of  oracle-jrockit-jvm_1.6.0-45-28.2.7_amd64.deb  is Architecture:amd64

**Comment by tumblingtumbleweed** [comment]
I am aware of that. The debian spec does not require a space after the colon, therefore this is a freight bug. Submitting a pull request now.

> The field ends at the end of the line or at the end of the last continuation line (see below). Horizontal whitespace (spaces and tabs) may occur immediately before or after the value and is ignored there; it is conventional to put a single space after the colon.

**Comment by rcrowley** [comment]
Closed by #39.
